packages/flutter/lib/src/rendering/wrap.dart

```diff
@@ -2,6 +2,7 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+import 'dart:developer';
```

Comment:
hopefully travis will catch this, but in case it doesn't, you probably want to remove this import

Reply:
Done